contrib/terraform/vsphere/modules/kubernetes-cluster/templates/metadata.tpl

```diff
@@ -0,0 +1,14 @@
+instance-id: ${hostname}
+local-hostname: ${hostname}
+network:
+  version: 2
+  ethernets:
+    ${interface_name}:
+      match:
+        name: ${interface_name}
+      dhcp4: false
+      addresses:
+        - ${ip}/${netmask}
+      gateway4: ${gw}
+      nameservers:
+        addresses: [${dns}]
```

Comment:
please add a new line, some linters complain about missing new line at the end of the file

Reply:
fixed in 6ff1febb48a69905ec6f67dd42fd44b8dd34062b